DEV: Add make-dev-page.py

ElDeveloper opened the pull request:
This creates a development page of Empress (optionally a tandem plot)
using the moving pictures tutorial data. To see changes from the
JavaScript and CSS files you can reload the generated page from the web
browser.

In the animated GIF below, I show how to see changes to the source by reloading the page (initially the log message is not visible in the console):

![dev-cycle](https://user-images.githubusercontent.com/375307/87369666-e9857e80-c535-11ea-8140-cc4900c907db.gif)


Fixes #242

kwcantrell commented:
@ElDeveloper thanks for adding this! I think this will help improve the development experience! Would it be possible to have the data set be a parameter? I have a couple datasets I like to test with.

ElDeveloper commented:
@kwcantrell, great suggestion that's very handy. I just added the CLI configuration to make this possible. If no arguments are set then the moving pictures dataset is loaded. I also added a couple of calls to .travis.yml so we check that this works as the code evolves.

kwcantrell commented on CONTRIBUTING.md:
```diff
@@ -30,6 +30,18 @@ make test
 If you just want to run the Python or JavaScript tests, you can run
 `make pytest` or `make jstest` respectively.
 
+## Front-end development
+
+For convenience, a utility script `./tests/python/make-dev-page.py` is bundled
+in the test suite. If no arguments are set, the script will load the moving
+pictures dataset and create a page `development-page.html`. Using this page
+developers can modify CSS and JS files and simply reload the page on the
+browser to see their changes take effect.
+
+Note, modifications to the Python code, or the template code require that you
+re-run the script. For a full list of options and arguments, you can run
+`./tests/python/make-dev-page.py --help`.
+
```
Can you modify this file to tell users to use pip install -e .

This script wasn't working for me because I used pip install .

kwcantrell commented:
Thanks @ElDeveloper

ElDeveloper commented:
Thanks @kwcantrell!

On (Jul-14-20|14:58), kwcantrell wrote:
>Merged #257 into master.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://urldefense.com/v3/__https://github.com/biocore/empress/pull/257*event-3546084126__;Iw!!Mih3wA!Wb5FJPQNDotwbVhcasV2u8cPh4dr1O6Qxqd2fURUTGzDlYX9l7EXiNz3VwRXiHs$
